cuqi/distribution/_cauchy.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+import scipy.stats as sps
+from cuqi.geometry import _get_identity_geometries
+from cuqi.utilities import force_ndarray
+from cuqi.distribution import Distribution
+
+class Cauchy(Distribution):
+    """ Multivariate Cauchy distribution of independent random variables.
+     
+    Each is distributed according to the PDF function
+
+    .. math::
+
+        \\frac{1}{\\pi\\gamma(1+\\frac{(x-\\mu)^2}{\\gamma^2})}
+
+    where :math:`\\mu` is the location parameter and :math:`\\gamma` is the scale parameter.
+
+    Parameters
+    ----------
+    location: float or array_like
+        Location parameter
+
+    scale: float or array_like
+        Scale parameter
+
+    Example
+    -------
+    .. code-block:: python
+
+        # % Generate a Cauchy distribution
+        import cuqi
+        location = 0
+        scale  = 1
+        x = cuqi.distribution.Cauchy(location, scale)
+        x.logpdf(0) # -1.1447298858494002
```

Comment:
Add example with 2 or 3 element location and scale to demonstrate being multivariate.

Reply:
Done